modules/number-picker/main/index.coffee

```diff
@@ -6,6 +6,34 @@ checkValue = (numberPicker, context) ->
   if min isnt undefined then value = Math.max(value, min)
   if value isnt oldValue then context.value(value)
 
+addHoldHandler = (incrementOnHold, incrementDelay, selection, incrementFn) ->
+  if incrementOnHold
+    holdStart = undefined
+    holdTimeout = undefined
+    incrementInterval = undefined
+
+    clearTimers = ->
+      clearTimeout(holdTimeout)
+      clearInterval(incrementInterval)
+
+    selection.on 'pointerdown', 'hx.number-picker', (e) ->
+      holdStart = Date.now()
+      document.activeElement.blur()
+      e.event.preventDefault()
+      fn = ->
+        incrementInterval = setInterval((-> incrementFn()), incrementDelay)
+      holdTimeout = setTimeout(fn, 200)
+
+    selection.on 'pointerup', 'hx.number-picker', ->
+      clearTimers()
+      if (Date.now() - holdStart) < 200
+        incrementFn()
+
+    selection.on 'pointerleave', 'hx.number-picker', clearTimers
```

Comment:
This resolves an issue with desktops when moving the mouse out of selection and releasing outside.
This is a safer way of doing it than adding listeners to the body